lib/ansible/modules/monitoring/zabbix/zabbix_host_triggers.py

```diff
@@ -0,0 +1,267 @@
+                  description: 'Process: NTPd is down'
+                  error: ''
+                  expression: "{13667}=0"
+                  flags: '0'
+                  last_event:
+                    acknowledged: '0'
+                    acknowledges:
+                    - alias: Admin
+                      clock: '1556182269'
+                      message: waiting customer action
+                    clock: '1556182854'
+                    eventid: '630'
+                    value: '1'
+                  lastchange: '1556182854'
+                  priority: '4'
+                  state: '0'
+                  status: '0'
+                  templateid: '13615'
+                  triggerid: '14040'
+                  type: '0'
+                  url: ''
+                  value: '1'
+'''
+
+DOCUMENTATION = '''
+---
+module: zabbix_host_triggers
+short_description: Get all triggers about a Zabbix host
+description:
+   - This module allows you to see if a Zabbix host have no active alert to make actions on it.
+     For this case use module Ansible 'fail' to exclude host in trouble.
+   - Length of "triggers_ok" allow if template's triggers exist for Zabbix Host
+version_added: "2.9"
+author:
+    - "Stéphane Travassac (@stravassac)"
+requirements:
+    - "python >= 2.7"
+    - "zabbix-api"
+    - "zabbix-api >= 0.5.3"
+options:
+    host_identifier:
+        description:
+            - Identifier of Zabbix Host
+        default: hostname
+        required: true
+    host_id_type:
+        description:
+            - Type of host_identifier
+        choices:
+            - hostname
+            - visible_name
+            - hostid
+        required: false
+        default: hostname
+    trigger_severity:
+        description:
+            - Zabbix severity for search filter
+        default: average
+        required: false
+        choices:
+            - not_classified
+            - information
+            - warning
+            - average
+            - high
+            - disaster
+    timeout:
+        description:
+            - timeout before fail
+        default: 10
+        required: false
+        type: int
+    validate_certs:
+        description:
+                - SSL validate certificate
+        default: true
+        required: true
+        type: bool
+extends_documentation_fragment:
+    - zabbix
+'''
+
+EXAMPLES = '''
+- name: exclude machine if alert active on it
+  zabbix_host_triggers_problem:
+      server_url: "{{ zabbix_url }}"
+      login_user: "{{ lookup('env','ZABBIX_USER') }}"
+      login_password: "{{ lookup('env','ZABBIX_PASSWORD') }}"
+      host_identifier: "{{inventory_hostname}}"
+      host_id_type: "hostname"
+      timeout: 120
+  register: zbx_host
+  delegate_to: localhost
+- fail:
+    msg: "machine alert in zabbix"
+  when: zbx_host['triggers_problem']|length > 0
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+try:
+    from zabbix_api import ZabbixAPI, ZabbixAPISubClass
```

Comment:
ZabbixAPISubClass import no longer needed
```suggestion
    from zabbix_api import ZabbixAPI
```